spec/helpers/formatDate.spec.js

```diff
@@ -0,0 +1,144 @@
+'use strict';
+const moment = require('moment');
+
+describe('Outlook Format Date', function test() {
+
+  const action = require('../../lib/processEventDataHelper');
+  const data = require('../data/processEventData_test.json');
+
+  //Check if the date FORMAT itself is as expected.
+  describe('formats input date string based on given parameter ', function test() {
+
+    let timeZone = 'Europe/Berlin';
+
+    it(' - YYYY-MM-DD - ISO Date, No UTC Offset', function test() {
+      let inputDate = '2016-12-19T18:00:00';
+      let format = 'YYYY-MM-DD';
+      let expectedOutput = format;
+      let actualOutput = action.formatDate(inputDate, timeZone, format);
+      expect(moment(actualOutput).creationData().format).toEqual(expectedOutput);
```

Comment:
expectation should check ``actualOutput``. We don't care about the format. Please change the expectation to following.

````js
expect(actualOutput).toEqual("2016-12-19");
````